hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/RatisPipelineProvider.java

```diff
@@ -59,7 +63,8 @@
 
   private final ConfigurationSource conf;
   private final EventPublisher eventPublisher;
-  private final PipelinePlacementPolicy placementPolicy;
+  //private final PipelinePlacementPolicy placementPolicy;
+  private PlacementPolicy placementPolicy = null;
```

Comment:
Keep it final and remove the commented line above.

Reply:
Made the change with,
`private PlacementPolicy placementPolicy = null;`
replaced with,
`private final PlacementPolicy placementPolicy;`